frontend/src/store/globals.ts

```diff
@@ -0,0 +1,37 @@
+import { ref } from 'vue';
```

Comment:
@aweebs Yes, if you trace performance in performance-intensive places (like going to library), you'll see fewer animation frames. I'm not sure if ``now`` was even garbage collected properly.

Why do you think it's strange?

Reply:
Just strange because they were already one liners so this isn't much clearer in terms of dev ex or readability. But if these are really improving performance in certain places than it makes sense.